Mathlib/Order/LatticeIntervals.lean

```diff
@@ -126,6 +126,16 @@ protected lemma isCompl_iff [Lattice α] [BoundedOrder α] {x y : Iic a} :
     IsCompl x y ↔ Disjoint (x : α) (y : α) ∧ (x : α) ⊔ (y : α) = a := by
   rw [_root_.isCompl_iff, Iic.disjoint_iff, Iic.codisjoint_iff]
 
+protected lemma complementedLattice_iff [Lattice α] [BoundedOrder α] :
```

Comment:
Are there similar results for other `Ixx`?

Reply:
There are, but looking closely, I see the theory of `Iic` is quite a bit more developed than the others already so I think it should be a a separate undertaking to even things out here.